CHANGES.rst

```diff
@@ -222,6 +231,10 @@ API Changes
 
 - ``astropy.vo``
 
+  - ``VOSDatabase.to_json()`` now accepts an ``overwrite`` argument.
```

Comment:
As above - if the argument has been renamed, make that clearer (at the moment it sounds like a new option has been added.